src/transformers/trainer.py

```diff
@@ -605,6 +608,19 @@ def _tune_save_checkpoint(self):
                 torch.save(self.optimizer.state_dict(), os.path.join(output_dir, "optimizer.pt"))
                 torch.save(self.lr_scheduler.state_dict(), os.path.join(output_dir, "scheduler.pt"))
 
+    def call_model_init(self, trial = None):
+        import six
```

Comment:
I'm not familiar with `six`. Is it in the standard library? Any reason not to use the `inspect` module instead?